package/CHANGELOG

```diff
@@ -13,6 +13,15 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+03/09/16 abhinavgupta94
```

Comment:
The CHANGELOG is used to collect all changes between releases. The version number is only updated when we do a release. In that time all changes are appended to the most recent entry in this file. This means you should update the 0.14.1 entry instead of creating a new one.